wpilibcExamples/src/main/cpp/examples/HAL/c/Robot.c

```diff
@@ -0,0 +1,123 @@
+/*----------------------------------------------------------------------------*/
+/* Copyright (c) 2018 FIRST. All Rights Reserved.                             */
+/* Open Source Software - may be modified and shared by FRC teams. The code   */
+/* must be accompanied by the FIRST BSD license file in the root directory of */
+/* the project.                                                               */
+/*----------------------------------------------------------------------------*/
+
+/*
+This example shows how to use the HAL directly, and what is needed to run a
+basic program. The sample is compiled in C, however the functionality works from
+C++ as well.
+
+The HAL is considered a stable but changeable API. The API is stable during a
+season, and is safe to use for events. However, between seasons there might be
+changes to the API. This is an advanced sample, and should only be used by users
```

Comment:
```suggestion
changes to the API. This sample demonstrates advanced programming, and should only be used by users
```

Reply:
I don't like the sound of that. Its not demonstrating advanced programming. Its an advanced sample.